release/scripts/release-pre.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euo pipefail
+
+echo $(pwd)
+
+VERSION_FILE=$(dirname  $(dirname $(dirname $0)))/VERSION
```

Comment:
This looks weird, as this intentional?

Reply:
Yes, it is unfortunately necessary because `cargo release` executes the hooks in the same folder as the crate it's releasing but we need to write to one VERSION file in the root of the project/workspace.